app/src/main/java/com/example/atheneum/fragments/OwnerPageFragment.java

```diff
@@ -59,11 +66,13 @@ public OwnerPageFragment() {
     @Override
     public void onStart() {
         super.onStart();
+        firebaseRecyclerAdapter.startListening();
     }
 
     @Override
     public void onStop() {
         super.onStop();
+        firebaseRecyclerAdapter.stopListening();
```

Comment:
I think you should stop listenening before calling super on stop maybe. Since you want to stop anything in your current activity before calling the base classes onStop.